server/plugin.go

```diff
@@ -340,6 +343,30 @@ func (p *Plugin) handleList(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (p *Plugin) handleCount(w http.ResponseWriter, r *http.Request) {
+	userID := r.Header.Get("Mattermost-User-ID")
+
+	countIssues, err := p.listManager.CountIssues(userID)
+	if err != nil {
+		p.API.LogError("Unable to get issues for user err=" + err.Error())
+		p.handleErrorWithCode(w, http.StatusInternalServerError, "Unable to get issues for user", err)
+		return
+	}
+
+	countIssuesJSON, err := json.Marshal(countIssues)
+	if err != nil {
+		p.API.LogError("Unable marhsal count issue list to json err=" + err.Error())
+		p.handleErrorWithCode(w, http.StatusInternalServerError, "Unable marhsal count issue list to json", err)
+		return
+	}
+
+	_, err = w.Write(countIssuesJSON)
+	if err != nil {
+		p.API.LogError("Unable to write json response err=" + err.Error())
+	}
+
+}
```

Comment:
Nit: extra newline at the end of this function and `listStore.CountIssues`. This is something we should have the linter configured to enforce, but I'm not sure if we have this particular convention enforced anywhere